.github/workflows/e2e-tests.yaml

```diff
@@ -23,9 +22,19 @@ jobs:
       tag: ${{ github.sha }}
 
   e2e-tests:
-    name: Execute E2E Tests
+    name: E2E tests with RING S3 v${{ matrix.ring_version }}
     runs-on: ubuntu-22.04-8core
     needs: dev-image
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - ring_version: "9.4"
+            github_varirable_name: "CLOUDSERVER_RING_9_4"
+          - ring_version: "9.5"
+            github_varirable_name: "CLOUDSERVER_RING_9_5"
+    env:
+      CLOUDSERVER_TAG: ${{ vars[matrix.github_varirable_name] }}
```

Comment:
This is getting more complicated then it should be, at this point I would start thinking about hardcoding it, script a way to retrieve that info.

Reply:
Yeah! but for now I want it to be configurable, I agree its complicated but not that much for now.